scripts/Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN apt-get update && apt-get install -y git build-essential pandoc vim ffmpeg \
 
 # Copy needed scripts
 COPY $SRC_DIR /opt/arviz/requirements*.txt
-COPY $SRC_DIR /opt/arviz/scripts
+COPY $SRC_DIR /opt/arviz
```

Comment:
But we don't want to import all at this point --> docker cache will fail everytime something is changed in the codebase

Reply:
ah i see. Ok let me try and figure this out

Reply:
Fixed